### ISSUE TITLE
Setup Tome Sync in a PR (Elijah)

### ISSUE THREAD

**Issue by brandonrapp** [department-of-veterans-affairs/va.gov-team#4876]
Setup Tome Sync in a PR (Elijah)
https://va-gov.atlassian.net/browse/VAGOV-6194 (moving from Jira to here)

- [ ] Figure out Drush 9 workaround on CMS-CI (DevShop), as it isn't supported yet, but only needs to be run manually for the initial export. Document process here. 
- [x] Get PR running that has the export directory exported with Drush 9
https://github.com/department-of-veterans-affairs/va.gov-cms/pull/827
- [x] Manually tar up for now and make available at /export/data.tar (need faster volume IO in #2984 to automate the tar on the publish event) 
- [ ] [EXTRA] Manually tar up sites/default/files (exclude js, css etc) and make available at /export/files.tar

**Comment by ElijahLynn** [comment]
We are moving from Jira so I am putting my notes here. We may end up using the CMS repo for these issues but they are moveable and copyable so....

**Comment by ElijahLynn** [comment]
> Actually in progress now, as of today.

For real now, tonight actually.

**Comment by brandonrapp** [comment]
@ElijahLynn - So the Chris and Brooks really need to get this moving. They would like an endpoint that gives them the most up to date content tarball. It would also need to have the meta tags included as well since they had those prior to starting their transformation work.

**Comment by ElijahLynn** [comment]
> @ElijahLynn - So the Chris and Brooks really need to get this moving. They would like an endpoint that gives them the most up to date content tarball. It would also need to have the meta tags included as well since they had those prior to starting their transformation work.

:+1: The other additions may be in this PR or multiple follow up PRs, not sure yet. But metatags, entity_id, breadcrumbs and "processed" field will be added after this part is done.

**Comment by KevinMHoffmanUSDS** [comment]
@ElijahLynn Hey there... when will the up-to-date tarball be completed? cc @brandonrapp @JeffBarnesUSDS @gunsch

**Comment by ElijahLynn** [comment]
> a

I had committed to mid-this week for that on one of our daily syncs. Let's shoot for Thursday (since yesterday was holiday).

**Comment by KevinMHoffmanUSDS** [comment]
@ElijahLynn Thanks!

**Comment by KevinMHoffmanUSDS** [comment]
@ElijahLynn Good morning! Any updates on what we discussed above for last Thursday, January 23rd? Looks like this issue is still open, but If I'm missing something or not aware of a dependency, please let me know. cc @brandonrapp @gunsch @jefflbrauer @JeffBarnesUSDS

**Comment by ElijahLynn** [comment]
> @ElijahLynn Good morning! Any updates on what we discussed above for last Thursday, January 23rd? Looks like this issue is still open, but If I'm missing something or not aware of a dependency, please let me know. cc @brandonrapp @gunsch @jefflbrauer @JeffBarnesUSDS

Still working on #4923. The others are done but this one is more challenging. Hoping to have this one done by Wednesday.

**Comment by KevinMHoffmanUSDS** [comment]
@ElijahLynn Thanks for the update!

**Comment by KevinMHoffmanUSDS** [comment]
From Slack, as per @jefflbrauer 

@JeffB just a heads up that the file export (tome sync) work @Elijah Lynn is doing is much more complicated than previously estimated. Additionally, our CI is down and so he is being pulled to work with Jon to fix it as that blocks the tome sync work. So - if we can get CI fixed today, Elijah then anticipates completing the file export work by Thursday eod. He is in communication with Brandon about this, but I want to make sure you are aware as well. cc @Clarence Maeng

**Comment by ElijahLynn** [comment]
Update for today is that I am still finishing up urgent CI work that I was working on yesterday. I _should_ be working on this again today but we may be pushing to Friday EOD. 

My remaining work is on the breadcrumb export, which is much more complicated than I had estimated. It isn't undoable, but just not a trivial thing to add as I expected. If I can get some serious focus time today I'll make good progress on it. 

Tech Notes:
I'm having challenges passing in the correct routeMatch object into the Breadcrumb Manager service as it is the correct interface but turns out to actually need to be a currentRouteMatch object, not just a routeMatch. And the currentRouteMatch is not valid because we have multiple entity updates (nodes and paragraph references) so currentRouteMatch isn't going to work, because the currentRouteMatch is actually the 'edit' route, not the view route. 

Further, the Breadcrumb builder doesn't actually appear to even use that object passed in, so I think I need to extend that class to make use of the object we pass.

**Comment by ElijahLynn** [comment]
Update: I finally got a chance to work on this again yesterday. Made progress. New ETA EOD tomorrow, and gonna ask a team member for some pairing on this to speed up. 

Tech Notes:
I got passed my above issue and am now on the issue of figuring out why they MenuBasedBreadcrumbBuilder isn't kicking in for my test node, and falling back to PathBasedBreadcrumbBuilder. I suspect it is something to do with the `context` that is being inherited, and haven't found out where yet.
